### ISSUE TITLE
When using a webcomponent, I want to be able to mark it as blocking loading per View, so I can have a fine control on how to optimize the loading process of my application.

### ISSUE THREAD

**Issue by gilberto-torrezan** [vaadin/flow#1503]
With the #1487 is possible to mark a webcomponent class as non-blocking loading. But when using different instances of the same webcomponent in different Views, you may want to change the default loading behavior depending on the context of each place.

Example: suppose you have a webcomponent that loads the title of the application. It appears in the front page and it should be loaded as soon as possible (blocking - so it gets the priority over non-blocking components). In this case the annotation should be `@HTMLImport(value = "mycomponent.html", blocking = true)`.

But if you use the same component in other View of the same application, and that View is not the first view of the system, the component should be loaded in a non-blocking way. In that case, the annotation should be `@HTMLImport(value = "mycomponent.html", blocking = false)`.

That's more complicated if you have an external component (one that you can't change the code) and you want to change how is it loaded (from blocking to non-blocking or the other way around).

In summary: this ticket is about a way to change how the component is loaded in runtime, by not just using the annotation value.
